code/modules/clothing/suits/f13armor.dm

```diff
@@ -1359,8 +1359,36 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	slowdown = 0.05
 
+//Fluff Stuff
 /obj/item/clothing/suit/armor/f13/rangercombat/finlayranger
 	name = "Brotherhood of Steel Ranger Armor"
 	desc = "A customized and well-worn suit of riot gear with parts of the suit reinforced with leather armor."
 	icon_state = "finlayranger"
 	item_state = "finlayranger"
+
+/obj/item/clothing/suit/armor/f13/rangercombat/finlayranger_alt
+	name = "worn ranger armor"
+	desc = "A customized and well-worn suit of riot gear with parts of the suit reinforced with leather armor."
+	icon_state = "reshiranger"
+	item_state = "reshiranger"
+
+/obj/item/clothing/suit/armor/f13/combat/swat/bibledog
+	name = "SLPD SWAT combat armor"
+	slowdown = 0.11
```

Comment:
You're keeping the armor value the same but lowering the slowdown by... 0.01. *Why?*